lib/active_force/association/eager_load_builder_for_nested_includes.rb

```diff
@@ -0,0 +1,84 @@
+module ActiveForce
+
+  module Association
+    class InvalidAssociationError < StandardError; end
+
+    class EagerLoadBuilderForNestedIncludes
+
+      class << self
+        def build(relations, current_sobject, parent_association_field = nil)
```

Comment:
Might just be me, but if there's only one class method, I think it's more readable to have `def self.build` instead of this `class << self` block.